lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -252,6 +257,26 @@ def active_record_model?(const)
           !const.abstract_class?
         )
       end
+
+      def pending_migrations_message
+        return unless defined?(ActiveRecord)
+
+        ActiveRecord::Migration.check_all_pending!
+        nil
+      rescue ActiveRecord::PendingMigrationError => e
+        e.message
+      end
+
+      def run_migrations
+        # Running migrations invokes `load` which will repeatedly load the same files. It's not designed to be invoked
+        # multiple times within the same process. To avoid any memory bloat, we run migrations in a separate process
+        stdout, status = Open3.capture2(
+          { "VERBOSE" => "true" },
+          "bundle exec rake db:migrate",
```

Comment:
I expect we should be using `rails db:migrate` to match the modern way.

(it's been available since https://github.com/rails/rails/pull/22288 so there shouldn't be any problem with backwards compatibility).

Reply:
Changed to `rails db:migrate` 👍